skimage/measure/_label.py

```diff
@@ -1,6 +1,32 @@
+from scipy import ndimage
 from ._ccomp import label_cython as clabel
 
 
+def _label_bool(image, background=None, return_num=False, connectivity=None):
+    """
+    Faster implementation of clabel for boolean input.
+    """
+    from ..morphology._util import _resolve_neighborhood
```

Comment:
:facepalm: again the circular import...
Not for this PR, but we may be should move `morphology._util` to `skimage.utils`.

Reply:
indeed :)